From 910609a451a87e5d711df1f35f19d46b727d17f9 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Mon, 7 Aug 2023 10:43:44 +0200
Subject: [PATCH] regress/key-options.sh: update future key expiry date to far
 in the future

This allows testing Y2038 with system time set to after that (i.e.2040),
so that actual Y2038 issues can be exposed, and not masked by key expiry time
errors.

Upstream-Status: Submitted [https://github.com/openssh/openssh-portable/pull/425 and https://bugzilla.mindrot.org/show_bug.cgi?id=3684]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 regress/key-options.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/regress/key-options.sh b/regress/key-options.sh
index 7f8166d..e17ff6a 100644
--- a/regress/key-options.sh
+++ b/regress/key-options.sh
@@ -120,5 +120,5 @@ check_valid_before() {
 check_valid_before "default"	""				"pass"
 check_valid_before "invalid"	'expiry-time="INVALID"'		"fail"
 check_valid_before "expired"	'expiry-time="19990101"'	"fail"
-check_valid_before "valid"	'expiry-time="20380101"'	"pass"
+check_valid_before "valid"	'expiry-time="25250101"'	"pass"
 
